LICENSE.md

```diff
@@ -67,58 +67,59 @@ The User may not limit access to the Mod in any way, except as provided by the G
 The User may decompile compiled binaries of the Mod.  
 The User may examine the Mod's source code.
 
-5. Right to distribute
+5. No right to distribute
 --------
 
-The Author reserves the right to distribute the Mod.  
-The Author may designate a person or organization as a Distributor.  
-A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
-The User may not distribute the Mod without being granted distribution rights from the Author.  
-The Author may revoke distribution rights from any Distributor.  
+The Author reserves the right to transmit verbatim copies of ("Distribute") the Mod.  
+Rights to distribute the Mod must be granted in a separate license which must be explicitly obtained from the Author.
+
+If a user wishes others to aquire the mod, the user may provide a link to the Author's website. This doesn't need to be granted as a right. We're just letting you know the acceptable way your friends may download the Mod.
 
 6. Right to contribute
 --------
 
-The User may "fork" the Codebase.  
+The User may copy ("fork") the Codebase.  
 The User may compile the Codebase.  
-The User may submit contributions to the Author for inclusion in the Mod.  
-The User grants the Author all rights to any contribution.  
-The User renounces all rights to their contribution except as specified under this license.  
-The User retains the right to re-use code they've written.  
-The Author may specify a Contributor License Agreement allowing a Contributor to retain some or all rights.  
+The User may submit source code or other materials (a Contribution) to the Author to be considered for inclusion in the Mod.  
+Unless specified by a seperate Contributor License Agreement, the Contributor agrees to the following terms.  
+If the contribution is accepted by the Author:
+ - Full copyright of the contribution is assigned to the Author.
+ - The author grants the Contributor a license to use, modify, and distribute the contribution.
 
-7. Right to derive
+7. Right to create and distribute Derivatives
 --------
 
-The User may create Derivative(s) based on the Mod.  
-A Derivative must contain changes which a reasonably informed person would consider significant.  
-
-The User may distribute a Derivative if the following conditions are met:
- - The Derivative does not generate revenue.
- - The Derivative provides credit to the Author.
- - The Derivative complies with the Mod license.
+The User may create and distribute a Derivative of the Mod if all of the following conditions are met:
```

Comment:
Nope nope nope. Combining the right to create and right to distribute like this is ignoring the differences between the two.

A User can create a derivative, without restriction. However, to distribute, they have to meet certain conditions. Your rework of this section merges these two, and creates extra limitations which ignore the point of this section.

Part of the intent here is that if the original author disappears, porting a mod to a new version of the game qualifies as a derivative. A dead mod can live again under a new maintainer.

Yes, this creates the possibility that users will like the derivative more than the original, and if a dead mod's author shows up again after a year, things may be a bit...interesting. But this prevents someone from generating revenue with a derivative, which allows the original author to reap the benefits of their work.